packages/plugin-doc-coverage/src/lib/runner/runner.unit.test.ts

```diff
@@ -0,0 +1,88 @@
+import type { CoverageResult } from './models.js';
+import { trasformCoverageReportToAudits } from './runner.js';
+
+describe('trasformCoverageReportToAudits', () => {
+  const mockCoverageResult = {
+    functions: {
+      coverage: 75,
+      nodesCount: 4,
+      issues: [
+        {
+          file: 'test.ts',
+          line: 10,
+          name: 'testFunction',
+          type: 'functions',
+        },
+      ],
+    },
+    classes: {
+      coverage: 100,
+      nodesCount: 2,
+      issues: [],
+    },
+  } as unknown as CoverageResult;
+
+  it('should transform coverage report to audit outputs with no filters', () => {
+    const result = trasformCoverageReportToAudits(mockCoverageResult, {});
+    expect(result).toMatchSnapshot();
```

Comment:
For readebilite and file switch i would not use a snapshot here. Same for other tests in this file

Reply:
I agree, I have redone them, let me know what do you think ^^